android/app/src/main/java/app/organicmaps/car/util/RoutingUtils.java

```diff
@@ -93,20 +92,13 @@ private static Step createNextStep(@NonNull final CarContext context, @NonNull R
     return builder.build();
   }
 
+  @SuppressWarnings("NewApi") // ZonedDateTime is backported for Android versions below 8.0.
   @NonNull
   private static TravelEstimate createTravelEstimate(@NonNull app.organicmaps.util.Distance distance, int time)
   {
-    final TravelEstimate.Builder builder = new TravelEstimate.Builder(RoutingHelpers.createDistance(distance), createTimeEstimate(time));
-    builder.setRemainingTimeSeconds(time);
-    builder.setRemainingDistanceColor(Colors.DISTANCE);
-    return builder.build();
-  }
-
-  @NonNull
-  private static DateTimeWithZone createTimeEstimate(int seconds)
-  {
-    final Calendar currentTime = Calendar.getInstance();
-    currentTime.add(Calendar.SECOND, seconds);
-    return DateTimeWithZone.create(currentTime.getTimeInMillis(), currentTime.getTimeZone());
+    return new TravelEstimate.Builder(RoutingHelpers.createDistance(distance), ZonedDateTime.now().plusSeconds(time))
```

Comment:
How did you find which methods should be replaced?

Reply:
I checked the code for `TravelEstimate.Builder`, and it included a constructor that accepted a `ZonedDateTime`: https://developer.android.com/reference/kotlin/androidx/car/app/navigation/model/TravelEstimate.Builder#Builder(androidx.car.app.model.Distance,java.time.ZonedDateTime)

Reply:
Did you test your changes, including AA, on devices older than API 26?

Reply:
I'm getting this list of errors when building: 
[Build error.txt](https://github.com/organicmaps/organicmaps/files/14502746/Build.error.txt)